docs/pages/development/installation.md

```diff
@@ -8,36 +8,36 @@ import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
 import { Tab, Tabs } from '~/components/plugins/Tabs';
 import { Terminal } from '~/ui/components/Snippet';
 
-The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or Bare project. To add a the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
+> Note: These docs assume a basic level of familiarity with React-Native. If you are new to React-Native, please see the [React-Native docs](https://reactnative.dev/docs/getting-started).
 
-If you're just starting your project, you can create a new project from our template with:
+The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or bare project. To add the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
 
-<Terminal cmd={["$ npx crna -t with-dev-client"]} />
 
-If you have an existing project, you'll need to [install the package and make a few changes](installation.md) to your **AppDelegate.m**, **MainActivity.java** and **MainApplication.java**.
+If you're just starting your project, you can run the following command to create a new project with Expo modules pre-installed: 
 
-If you created your project with `expo init` or `create-expo-app`, or you already have `expo` and/or other Expo modules up and running, use the tabs marked **With Expo modules**.
+<Terminal cmd={["$ npx crna -y myapp"]} />
```

Comment:
```suggestion
<Terminal cmd={["$ npx create-expo-app my-app"]} />
```
Shouldn't we recommend `create-expo-app` instead? Alternate form will be `yarn/npm create expo-app`.